### PR TITLE
Pods 5782

### PR DESCRIPTION
**Pull request by jigneshkumarpatel**

### CODE REVIEW

File: app/identifiers/trustees/partnership/contact/EnterEmailId.scala [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+case class EnterEmailId(index: Int) extends TypedIdentifier[String] {
+  override def path: JsPath =
+    TrusteesId(index).path \ EnterEmailId.toString
+}
+
+object EnterEmailId {
+  override lazy val toString: String ="partnershipEmail"
+
+  def collectionPath(index: Int): JsPath =
```
Comment (madhuranakate) [inline app/identifiers/trustees/partnership/contact/EnterEmailId.scala:31]: is this used anywhere? we can get rid of the collectionPath if not used

File: app/identifiers/trustees/partnership/contact/EnterPhoneId.scala [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+case class EnterPhoneId(index: Int) extends TypedIdentifier[String] {
+  override def path: JsPath =
+    TrusteesId(index).path \ EnterPhoneId.toString
+}
+
+object EnterPhoneId {
+  override lazy val toString: String ="partnershipPhone"
+
+  def collectionPath(index: Int): JsPath =
```
Comment (madhuranakate) [inline app/identifiers/trustees/partnership/contact/EnterPhoneId.scala:31]: same here

File: app/helpers/cya/trustees/partnership/TrusteeContactDetailsCYAHelper.scala [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      Some(answerOrAddRow(
+        id = EnterPhoneId(index),
+        message = Message("messages__enterPhone_cya_label", trusteePartnershipName).resolve,
+        url = Some(routes.EnterPhoneController.onPageLoad(index, CheckMode).url),
+        visuallyHiddenText = Some(msg"messages__enterPhone__cya__visuallyHidden".withArgs(trusteePartnershipName))
+      ))).flatten
+
+    rowsWithDynamicIndices(rowsWithoutDynamicIndices)
+  }
+}
```
Comment (madhuranakate) [inline app/helpers/cya/trustees/partnership/TrusteeContactDetailsCYAHelper.scala:64]: Tests should be added for CYA helper, spoke creation and data completion

**Review by madhuranakate — CHANGES_REQUESTED** [review]

File: app/helpers/SpokeCreationService.scala [hunk 4]
```diff
@@ -22,6 +22,7 @@ import helpers.spokes.establishers.individual._
 //import helpers.spokes.establishers.partnership.{EstablisherPartnershipAddress, EstablisherPartnershipDetails, EstablisherPartnerDetails}
```
Comment (grantrobson) [inline app/helpers/SpokeCreationService.scala:22]: Commented line

**Review by grantrobson — APPROVED** [review]

**Pull request merged**